compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/PlatformWindowContext.uikit.kt

```diff
@@ -65,6 +77,36 @@ internal class PlatformWindowContext {
         )
     }
 
+    /**
+     * Converts the given [Rect] from the coordinate space of the container window to the coordinate
+     * space of the root UIWindow in which hierarchy with the container window resides.
+     */
+    fun convertContainerWindowRectToRootWindowCGRect(rect: Rect): CValue<CGRect> {
+        val windowContainer = _windowContainer ?: return CGRectZero.readValue()
+
+        val windowContainerWindow = windowContainer.window
+
+        fun UIWindow.density(): Density {
```

Comment:
no need to be local function, move to the end of the file

Reply:
I don't want to move it in that scope, because that's not a `real` density with correct `fontScale`. It's only there because following calculations need `Density` instead of just `Float`, and that can be derived from `UIWindow` directly.